azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/model/implementation/CreatableImpl.java

```diff
@@ -33,32 +33,6 @@ protected CreatableImpl(String name, InnerModelT innerObject) {
     }
 
     /**
-     * create this resource and creatable resources it depends on.
```

Comment:
minor stylistic issue: we start each description sentence in javadocs capitalized (and period at the end) -- as it is  a proper sentence.
It's only in the param and return descriptions that we start with small letters and no period (because those typically are not full sentences)